interface/skin.dmf

```diff
@@ -49,13 +50,12 @@ menu "menu"
 window "mainwindow"
 	elem "mainwindow"
 		type = MAIN
-		pos = 0,0
+		pos = 281,0

```

Comment:
what the heck is this?

Reply:
Don't know what's with this, DM just automatically did this? I didn't edit the main window =|

Reply:
well dont' commit changes you didn't make dummy, use git add -p to add patch by patch

Reply:
or just use gitkraken